apps/lndg/db.sqlite3

```diff
@@ -0,0 +1 @@
+
```

Comment:
Does this file need to be manually created? Maybe it would be cleaner to pass in an empty dir and LNDg would create the DB file if it doesn't exist.

Reply:
Thanks for the comment, that is indeed is a bit cleaner approach.  I made a few changes to achieve this and uploaded a new image. This branch is now up to date as well for the change.